src/App.js

```diff
@@ -1,23 +1,53 @@
-import logo from './logo.svg';
-import './App.css';
+import "./App.css";
+import { Route, NavLink, Switch } from "react-router-dom";
+import Menu from "./components/Menu";
+import Order from "./components/Order.js";
+import Bill from "./components/Bill.js";
+import Service from "./components/Service.js";
 
 function App() {
   return (
     <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
+      <header className="app__header">
+        <Switch>
+          <Route path="/menu">
+            <Menu />
+          </Route>
+          <Route path="/order">
+            <Order />
+          </Route>
+          <Route path="/bill">
+            <Bill />
+          </Route>
+          <Route path="/service">
+            <Service />
+          </Route>
+        </Switch>
```

Comment:
I think your "/" path is missing here, right? where does the user end if they enter the root path?

Reply:
you're right :D I forgot. Thanks!